jsonrpc.js

```diff
@@ -536,8 +628,9 @@ module.exports = async function create({id, socket, channel, logLevel, logger, m
                             }
                         }));
                     } else if (body && body.result !== undefined && body.error === undefined) {
-                        if (/\.service\.get$/.test($meta.method)) Object.assign(body.result[0], requestParams);
```

Comment:
I am not sure about changing `result[0]` with `result` but it was committed this way before my changes. Please advice